helpers.py

```diff
@@ -39,7 +45,7 @@ def get_nested(meter_data, path):
                 value = 0
     return value
 
-def get_ahoy_field_by_name(meter_data, actual_inverter, fieldname, ch0_fld_names = True):
+def get_ahoy_field_by_name(meter_data, actual_inverter, fieldname, use_ch0_fld_names = True):
     '''get the value by name instead of list index'''
```

Comment:
Please add "...if use_ch0_fld_names is true, then the field names from the ch0_fld_names section in the JSON is used instead of the channel related fld_names..." (please correct if wrong.)

Reply:
Added a proper comment for explanation